tests/system/csv-import.rb

```diff
@@ -27,4 +27,44 @@
             row = $database.getRow('user', 'prueba3@hotmail.com', 'email')
             (row['name']).should.equal('maxi')
         end
+
+        it 'should fail if the uploaded file is not a csv' do
+
+            result= request('/system/csv-import', {
+                csrf_userid: $csrf_userid,
+                csrf_token: $csrf_token,
+                password: $staff[:password],
+                file: File.open( "../server/files/pdf-test.pdf")
+            })
+
+            (result['status']).should.equal('fail')
+            (result['message']).should.equal('INVALID_FILE')
+
+        end
+
+        it 'should upload a csv file' do
+
+            result= request('/system/csv-import', {
+                csrf_userid: $csrf_userid,
+                csrf_token: $csrf_token,
+                password: $staff[:password],
+                file: File.open( "../server/files/test.csv")
```

Comment:
The file `.gitignore` has an entry for `server/files/`, so, I see there's no new file `test.csv` in this PR.

The thing is, I'm sure you have this file locally, and your tests run locally, but since the file `test.csv` is not committed to the repo, it won't work. I think you have to make an exception on this file (and the `pdf-test.pdf` as well) so that it _does_ get committed into the repo. You should check the guides on gitignore to achieve this.